src/cli/README.md

```diff
@@ -1701,6 +1702,43 @@ Set the transmit power.
 Done
 ```
 
+### unsecureport add \<port\>
+
+Add a port to the allowed unsecured port list.
+
+```bash
+> unsecureport add 1234
+Done
+```
+
+### unsecureport remove \<port\>
+
+Remove a port from the allowed unsecured port list.
+
+```bash
+> unsecureport remove 1234
+Done
+```
+
+### unsecureport remove all
+
+Remove all ports from the allowed unsecured port list.
+
+```bash
+> unsecureport remove 1234
```

Comment:
nit: example here doesn't match the command to illustrate.

btw, how about renaming it as `unsecureport clear`?

Reply:
The function name is `otIp6RemoveUnsecurePort`, I think it is better to use the command `unsecureport remove all`.